lms/djangoapps/discussion_api/api.py

```diff
@@ -487,7 +488,7 @@ def _check_editable_fields(cc_content, data, context):
     )
 
 
-def _do_extra_actions(api_content, cc_content, request_fields, actions_form, context):
+def _do_extra_actions(api_content, cc_content, request_fields, actions_form, context, request):
```

Comment:
Lets retain that existing behaviour by replacing the `assert` with a `KeyError`.